### PR TITLE
LibreRead

### PR DESCRIPTION
**Pull request by cavebeat**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines. 
        
    ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language` ``

- [x] Your additions are ordered alphabetically.
- [x] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md).
- [x] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [x] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant issues or PRs.
- [x] Any category you are creating has the minimum requirement of 3 items.

related https://github.com/Kickball/awesome-selfhosted/issues/1077
put it into Category misc, because related to COPS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -793,6 +793,7 @@ See also <https://en.wikipedia.org/wiki/List_of_streaming_media_systems>, <https
   * [How Secure Is My Password](https://github.com/howsecureismypassword/hsimp) - Rather than just saying a password is "weak" or "strong", How Secure is My Password? lets your users know how long it would take someone to crack their password. ([Demo](https://howsecureismypassword.net/)) `MIT` `Javascript`
   * [google-webfonts-helper](https://github.com/majodev/google-webfonts-helper) `⚠` - A Hassle-Free Way to Self-Host Google Fonts. Get eot, ttf, svg, woff and woff2 files + CSS snippets. ([Demo](https://google-webfonts-helper.herokuapp.com/)) `MIT` `Nodejs`
   * [King Phisher](https://github.com/securestate/king-phisher) - King Phisher is a tool for testing and promoting user awareness by simulating real world phishing attacks. `BSD` `Python`
+  * [LibreReader](https://savannah.nongnu.org/projects/libreread/) - Free(Libre) Ebook reader. ([Demo](http://demo.libreread.org/), [Source Code](https://savannah.nongnu.org/git/?group=libreread)) `GPLv3` `Go`
```
Comment (nodiscc) [inline README.md]: Demo returns error 502 Bad Gateway

**Comment by cavebeat** [comment]
@nodiscc please check "now", demo is working at the moment ;)

**Comment by nodiscc** [comment]
@cavebeat the credentials for the demo instance are not available anywhere, and the register option doesn't appear to work (still waiting for that confirmation email)

**Comment by Kickball** [comment]
@nodiscc @cavebeat I am getting a 502 error with the demo again.

**Comment by nodiscc** [comment]
No updates for one month, closing.

**Pull request closed without merging**